.github/workflows/issues_and_pull_requests.yml

```diff
@@ -12,6 +12,10 @@ env:
   DAYS_BEFORE_PR_CLOSE: 14
   DAYS_BEFORE_PR_STALE: 90
   EXEMPT_MILESTONES: "long-term-ideas"
+  STALE_CLARIFICATION: |
+    The purpose of this action is to enforce backlog review once in a while. This is mostly for maintainers and helps with keeping repository in good condition, because stale issues and PRs can accumulate over time and make it harder for others to find relevant information. It is also possible that some changes has been made to the repo already, and issue or PR became outdated, but wasn't closed for some reason. This action helps with periodic review and closing of such stale items in automated way.
+
+    You may let maintainers handle this or verify current relevancy by yourself, to help with re-triage. Any activity will remove stale label so it won't be automatically closed at this point.
```

Comment:
can we add one more paragraph, after the issue/pr got closed by bot? sth like "if you found this issue/PR after it's already closed, it doesn't mean that the idea got rejected - it's simply didn't got any priority for way too long to keep it open. If you would like to support it and make a PR, you are welcome to do so."

Reply:
Yes, we need to configure [`close-issue-message`](https://github.com/actions/stale#close-issue-message) and [`close-pr-message`](https://github.com/actions/stale#close-pr-message), I'll add it.

Reply:
Done in [`0061b57`](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7236/commits/0061b57d492b5a2730ec1f06fc775611d6e474c0).